File: umetnik.js
export class Umetnik {
    constructor(id, ime, mestoRodjenja, godinaRodjenja, brojDela) {
        this.id = id;
        this.ime = ime;
        this.mestoRodjenja = mestoRodjenja;
        this.godinaRodjenja = godinaRodjenja;
        this.brojDela = brojDela;   //broj dela koji se izlazu
    }

    updateDela(inc) {
        if(inc == 1)
            this.brojDela++;
        else if(inc == 0)
            this.brojDela--;
    }
}

File: API/Models/Umetnik.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace API.Models
{
    public class Umetnik
    {
        [Key]
        [Column("ID")]
        public int ID {get; set; }

        [Column("Ime")]
        [MaxLength(255)]
        public string Ime {get; set; }

        [Column("MestoRodjenja")]
        [MaxLength(255)]
        public string MestoRodjenja { get; set;}

        [Column("GodinaRodjenja")]
        [MaxLength(4)]
        public string GodinaRodjenja {get; set;}

        [Column("BrojDela")]
        public int BrojDela {get; set; }
    }
}